Introduce more guidance on getting start contributing (#523)
cncf/tag-app-delivery: website/content/en/blog/contributing-to-wg-platforms.md

---
title:  Getting started with contributing in WG Platforms
slug:   contributing-to-wg-platforms
date:   2023-12-22 12:00:00 +0000

> author: Abby Bangser & Atul Sharma

Reviewer: ```suggestion
author: Abby Bangser & Atulpriya Sharma
```

* Technical overview or experiences working with platform tools and related technologies
* Experience report or interviews about platform building or using
thought leadership in regards to supporting developer experience and * productivity
* Hands-on DIY posts helping readers learn a tool

Please keep in mind this is not an exhaustive list and are very open to new ideas. It may be easier to enumerate what is not fit for the working group:
* Vendor or other promotional pitches
* Topics not related to application delivery or cloud native technologies
* Discriminatory or abusive content

## Step 1 - Submission

We would encourage you to generate a GitHub issue and open a Slack thread with your idea. You can use [this link](https://github.com/cncf/tag-app-delivery/issues/new?template=community-contribution.md) to create an issue and also start a thread on the [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) in the [#wg-platforms channel](https://cloud-native.slack.com/archives/C020RHD43BP).

> This will present you with a templated issue where you can follow the prompts. You will first need to write a descriptive title, then answer each of the questions in the description field. If you find one is not relevant that is OK, just write that and there is always a chance to chat more on these things after your initial submission.

Reviewer: ```suggestion
The GitHub issue will present you with a template where you can follow the prompts. First, You must write a descriptive title, then answer each question in the description field. If you find one is not relevant that is OK, write that and there is always a chance to chat more on these things after your initial submission.

For Slack, you can start a new thread in this format: [Proposal]<My_New_Idea>
```

Once you’ve finished drafting your blog post, you can update the GitHub issue saying it’s ready for review as well as update the Slack with the draft as a Google Doc and tag anyone who you feel can review it. (PS: It’s an open forum so anyone is free to review, but if you feel there’s someone who must have a look at it, tag them)

During the review process, we’ll do the following checks:
* _Basic grammar, syntax, and language check_ - we suggest using a tool like Grammarly before submitting for review.
* _Technical correctness_ - we’ll validate the technical accuracy of what you’ve written.
* _Vendor Pitches/Promotional links_ - we’ll carefully go through the content to ensure there’s no promotional material.

## Step 4 - Final Approval and Publishing

After the review is complete, and there’s a consensus from everyone that this is good to go, the WG lead will initiate the process of raising a PR and merging it.

> What next?

Reviewer: ```suggestion
## What next?
```
